tests/components/acmeda/test_config_flow.py

```diff
@@ -39,7 +40,7 @@ async def async_generator(items):
         yield item
 
 
-async def test_show_form_no_hubs(hass, mock_hub_discover):
+async def test_show_form_no_hubs(hass: HomeAssistant, mock_hub_discover) -> None:
```

Comment:
Partial typed, which is not something we generally allow

Reply:
I was under the impression that the more type hints the better, and that partial was better than no type hints at all.

I wouldn't have added a new test partially typed, but since these are existing tests I thought it was better than the current.

Reply:
> I was under the impression that the more type hints the better, and that partial was better than no type hints at all.

Agree, I was reflecting the general policy we have on Core. I've just discussed with Martin, Erik & Paulus, and we agreed to allow partial typing for tests.

So feel free to revert and ignore my comment as of now 👍 

👍

Reply:
Great. I took out some partial hints out, now I've put them back in.

Ready for review again...